src/components/atoms/ChatMessage/ChatMessage.scss

```diff
@@ -1,82 +1,136 @@
 @import "scss/constants";
 
-$counterparty-text-border-radius: 28px 28px 28px 4px;
-$my-text-border-radius: 28px 28px 4px 28px;
+$counterparty-bottom-border-radius: 0 0 $spacing--xxl $spacing--xs;
+$counterparty-text-border-radius: $spacing--xxl $spacing--xxl $spacing--xxl
+  $spacing--xs;
+$my-text-bottom-border-radius: 0 0 $spacing--xs $spacing--xxl;
+$my-text-border-radius: $spacing--xxl $spacing--xxl $spacing--xs $spacing--xxl;
```

Comment:
Note: The reason to have `$counterparty-bottom-border-radius` and `$my-text-bottom-border-radius` is explained below